app/frontend/actions.js

```diff
@@ -60,13 +63,50 @@ module.exports = ({setState, getState}) => {
     )
   }
 
+  const handleError = (errorName, e, options) => {
+    if (e && e.name) {
+      // is a error
+      debugLog(e)
+      captureBySentry(e)
+      setState({
+        [errorName]: {
+          code: e.name,
+          params: {
+            message: e.message,
+            ...options,
+          },
+        },
+        error: e,
+      })
+    } else {
+      setState({
+        [errorName]: e,
+      })
+    }
+  }
+
   const setAuthMethod = (state, option) => {
     setState({
       authMethod: option,
       showExportOption: option === 'mnemonic' || option === 'file',
     })
   }
 
+  const fetchConversionRates = async (conversionRates) => {
+    try {
+      setState({
+        conversionRates: await conversionRates,
+      })
+    } catch (e) {
+      debugLog('Could not fetch conversion rates.')
+      setState({
+        conversionRates: null,
+      })
+      throw NamedError('ConversionRatesError', '`Could not fetch conversion rates.')
+    }
+    return true
```

Comment:
I don't see any caller function use this 'true' return value

Reply:
Thanks, you are right, there is no use for returning true.